<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Browser</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="backBtn">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../../resources/nicos-gui.qrc">
         <normaloff>:/previous</normaloff>:/previous</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonIconOnly</enum>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="fwdBtn">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../../../resources/nicos-gui.qrc">
         <normaloff>:/next</normaloff>:/next</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeBtn">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../../resources/nicos-gui.qrc">
         <normaloff>:/quit</normaloff>:/quit</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWebView" name="webView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="url">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../../resources/nicos-gui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>fwdBtn</sender>
   <signal>clicked()</signal>
   <receiver>webView</receiver>
   <slot>forward()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>84</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>backBtn</sender>
   <signal>clicked()</signal>
   <receiver>webView</receiver>
   <slot>back()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>23</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>26</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
